subworkflows/local/input_check.nf

```diff
@@ -4,16 +4,25 @@
 //
 
 include { SAMPLESHEET_CHECK } from '../../modules/local/samplesheet_check'
+include { RENAME_READS      } from '../../modules/local/rename_reads'
 
 workflow INPUT_CHECK {
     take:
     samplesheet // file: /path/to/samplesheet.csv
 
     main:
-    SAMPLESHEET_CHECK ( samplesheet )
-        .csv
+    ch_parsed_reads =
+        SAMPLESHEET_CHECK ( samplesheet ).csv
         .splitCsv ( header:true, sep:',' )
         .map { create_fastq_channel(it) }
+
+    if (params.aligner == 'cellrangermulti') {
+        ch_renamed_reads = RENAME_READS ( ch_parsed_reads ).reads
```

Comment:
@fmalmeida, this doesn't work as expected for me. It doesn't treat read pairs as pairs and therefore overwrites files with the same name. It should probably be run after the `groupTuple`, or even better, in the cellranger multi subworkflow. 

We can also skip renaming initially and revisit this once everything else works.

Reply:
I will re-check. Maybe putting it in the cellranger multi part, or module.
Because renaming for this sub-workflow would be a must, otherwise the tools do not find the reads.

Reply:
Unless they are already named correctly? Isn't this the same for standard cellranger?

Reply:
I will re-check. Because I think I had to do it for the "testing dataset". I will check it again during this week the reason why I added.
:)

Reply:
This was done, and the renaming operation was integrated in the module itself, with a skipping parameter to avoid doing so when desired.